rholang/src/main/scala/rholang/interpreter/RholangADT.scala

```diff
@@ -0,0 +1,92 @@
+// Normalization:
+// Normalization requires the evaluation of everything that doesn't go through
+// the tuplespace, so the top level of a normalized process must have everything
+
+package coop.rchain.interpreter
+
+case class Par(
+  sends: List[Send],
+  receives: List[Receive],
```

Comment:
I presume contracts are classified as receives and invocations as sends?

Reply:
Yes.